package.json

```diff
@@ -14,8 +14,11 @@
     "posttest": "npm stop && istanbul report --root coverage text-summary && istanbul check-coverage --lines 95 --statements 95 --branches 95 --functions 95",
     "test:watch": "karma start karma.local.js --auto-watch",
     "test:unit": "karma start karma.local.js --single-run",
-    "test:ci": "karma start karma.ci.js --single-run",
-    "test:image-diff": "./node_modules/.bin/mocha --compilers js:babel-core/register --ui tdd --timeout 15000 test/screenshot/diff-suite.js"
+    "test:unit-ci": "karma start karma.ci.js --single-run",
+    "test:ci": "./scripts/run-ci-tests.sh",
```

Comment:
I dont htink you'll need test:ci if you split up the travis.yml into two jobs

Reply:
👍